src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -151,6 +151,13 @@ public static class Font
 			/// Ignores text scale factor, resulting in a font size as dictated by the control.
 			/// </summary>
 			public static bool IgnoreTextScaleFactor { get; set; } = false;
+
+#if __MACOS__
+			/// <summary>
+			/// Ratio of the font-size vs logical pixel size
+			/// </summary>
+			public const float FontSizeRatio = 1.5f;
```

Comment:
Is there any basis for this number? Any source (docs, design guidelines etc) that can be linked to here?

Like, why is it weirdly small? What do native macOS apps do?

Reply:
Nope... but it's not giving an approximative result, it's giving EXACTLY the right result compared to UWP. I did a quick search and didn't found the explanation, that's why I created a configuration for this.

Reply:
@davidjohnoliver I found a better fix ;-)